---
source: tests/sentry/grouping/test_variants.py
---
app:
  hash: null
  component:
    app (exception of system takes precedence)
      chained-exception (ignored because hash matches system variant)
        exception*
          type*
            "MyApp.Exception"
          value*
            "Test 2"
        exception*
          type*
            "MyApp.Exception"
          value*
            "Test 1"
        exception*
          type*
            "System.AggregateException"
          value*
            "One or more errors occurred."
--------------------------------------------------------------------------
system:
  hash: "b4cd3ea9b8777433e90ffecbb9601091"
  component:
    system*
      chained-exception*
        exception*
          type*
            "MyApp.Exception"
          value*
            "Test 2"
        exception*
          type*
            "MyApp.Exception"
          value*
            "Test 1"
        exception*
          type*
            "System.AggregateException"
          value*
            "One or more errors occurred."
